CargadorDePeliculas.cpp

```diff
@@ -0,0 +1,102 @@
+#include "CargadorDePeliculas.h"
+
+/*1)*/CargadorDePeliculas::CargadorDePeliculas()
+{
+    //ctor
+    numeroPelicula=0;
+}
+
+/*2)*/CargadorDePeliculas::~CargadorDePeliculas()
+{
+    //dtor
+}
+
+/*3)*/void CargadorDePeliculas::cargar(string camino, Lista<*Pelicula> &cartelera){
+    /*
+    Funciona creando uno a uno objetos Pelicula, alojados en memoria dinamica, y cargandolos desde un archivo,
+    e introduciendo el puntero que los se�alaal final de Lista<*Pelicula>
+    */
+    //NOTA: El metodo fue adaptado de otro que cargaba un objeto Pelicula en lugar de un puntero. keep an eye for gramar erros
+
+    //Variables auxiliares de carga
+    Pelicula *peli;
+    string lectura;
+    int puntuacion;
+
+    //Abre el archivo
+    arch.abrirArchivo(camino);
+    numeroPelicula=0;
+
+    while(!arch.finalArchivo()){
+        //Crea un nuevo objeto en la memoria dinamica
+        peli = new Pelicula;
+
+        //Carga Titulo
+        lectura=arch.leerLinea();//Lee el archivo
+        peli->cargarTitulo(lectura);//carga en peli
+
+        //Carga Puntuacion
+        lectura=arch.leerLinea();//Lee
+        puntuacion=stoi(lectura);//"castea" de string a int
+        peli->cargarPuntaje(puntuacion);//carga en peli
+
+        //Carga Genero             el resto son iguales que el primero
+        lectura=arch.leerLinea();
+        peli->cargarGenero(lectura);
+
+        //carga en Director
+        lectura=arch.leerLinea();
+        peli->cargarDirector(lectura);
```

Comment:
Por algún motivo que desconozco, acá tuve que agregar una lectura antes de cargar el director
```suggestion
        lectura = arch.leerLinea();
        lectura = arch.leerLinea();
        peli->cargarDirector(lectura);
```
Porque cuando lo probé no funcionaba, y puse un cout de la lectura para ver que pasaba y se imprimía una linea en blanco a continuación de `Director: `. Si alguno logra darse cuenta por qué pasa eso y cómo se puede solucionar avisen!